src/Illuminate/Console/Command.php

```diff
@@ -117,9 +118,19 @@ public function run(InputInterface $input, OutputInterface $output)
             OutputStyle::class, ['input' => $input, 'output' => $output]
         );
 
-        return parent::run(
-            $this->input = $input, $this->output
-        );
+        try {
+            return parent::run(
+                $this->input = $input, $this->output
+            );
+        } catch (\Symfony\Component\Console\Exception\RuntimeException $e) {
+            $message = $e->getMessage();
+            if (Str::startsWith($message, 'Not enough arguments')) {
```

Comment:
Doesn't Symfony's error handler always print usage whenever any RuntimeException is thrown. Why don't we just do that here, too, regardless of the content of the exception?

Maybe I'm wrong here. Can you check?

Reply:
I think you are right.

In Symfony\Console\Application:

```php
 public function renderThrowable(\Throwable $e, OutputInterface $output): void
    {   
        $output->writeln('', OutputInterface::VERBOSITY_QUIET);

        $this->doRenderThrowable($e, $output);

        if (null !== $this->runningCommand) {
            $output->writeln(sprintf('<info>%s</info>', sprintf($this->runningCommand->getSynopsis(), $this->getName())), OutputInterface::VERBOSITY_QUIET);
            $output->writeln('', OutputInterface::VERBOSITY_QUIET);
        }
    }
```


While looking, I did see that there are other cases where Symfony throws a RunTime exception that we probably would want to print usage (such as wrong options, missing option values, etc)

I'll adjust the PR accordingly.